exasol/analytics/query/handler/udf/interface.py

```diff
@@ -0,0 +1,23 @@
+from exasol.analytics.query.handler.query_handler import QueryHandler
+
+
+from abc import ABC, abstractmethod
+
+from exasol.analytics.query.handler.context.scope import     ScopeQueryHandlerContext
+
+
+class UDFQueryHandler(QueryHandler[str, str]):
```

Comment:
```suggestion
class Handler(QueryHandler[str, str]):
```

Reply:
The current PR only renames files and directories. 
I propose to rename classes in the scope of the separate issue #211.